config/crd/bases/anywhere.eks.amazonaws.com_clusters.yaml

```diff
@@ -250,6 +250,29 @@ spec:
                 type: array
               kubernetesVersion:
                 type: string
+              machineHealthCheck:
+                description: MachineHealthCheck allows to configure timeouts for machine
+                  health checks. Machine Health Checks are responsible for remediating
+                  unhealthy Machines. Configuring these values will decide how long
+                  CAPI would wait to remediate unhealthy machine or determine health
```

Comment:
Can we not mention CAPI? It adds a foreign concept to the API which IMO increases the level of entry to understand it. I think we should be able to define what this does in a CAPI agnostic way.